mec/beamline.py

```diff
@@ -32,25 +32,26 @@ def lights_off():
     target_x = epics_motor.IMS('MEC:USR:MMS:17', name='target x')
 
 
-#with safe_load('target hexapod'):
-#    from .devices import PI_M824_Hexapod 
-#    tc_hexapod = PI_M824_Hexapod('MEC:HEX:01', name='target hexapod') 
-#
-#    from .devices import TargetStage
-#    target = TargetStage(target_x, tc_hexapod, 0.75, 0.75)
-
-with safe_load('target xy stage'):
-    from .devices import TargetXYStage
-    from pcdsdevices import epics_motor 
-    target_y = epics_motor.Newport('MEC:PPL:MMN:09', name='target y')
-    tgt = TargetXYStage(target_x, target_y, 3.7, 3.5)
+with safe_load('target hexapod'):
+    from .devices import PI_M824_Hexapod 
+    tc_hexapod = PI_M824_Hexapod('MEC:HEX:01', name='target hexapod') 
+
+    from .devices import TargetStage
+    target = TargetStage(target_x, tc_hexapod, 0.75, 0.75)
+
+#with safe_load('target xy stage'):
```

Comment:
Is this the relic stage stack from when the target hexapod was out of commission for repairs?

Reply:
That looks like the target stack from when the hexapod was functional.